resources/views/vendor/apidoc/partials/info.blade.php

```diff
@@ -43,6 +43,8 @@
 
 # Authentication
 
+Routes marked with this label <span class="authenticated">Requires authentication</span> require authentication.
```

Comment:
There's still confusion whether "authentication" here means actual user (authorization grant etc) or just client grant is enough.